source/Radar.cpp

```diff
@@ -145,6 +148,7 @@ const Color &Radar::GetColor(int type)
 		*GameData::Colors().Get("radar anomalous"),
 		*GameData::Colors().Get("radar blink"),
 		*GameData::Colors().Get("radar viewport")
```

Comment:
```suggestion
		*GameData::Colors().Get("radar viewport"),
```
dont forget the comma ;)

Reply:
Yep. That, at least, I managed to figure out from the debug... Thanks!